tests/unit/indexer.test.ts

```
@@ -0,0 +1,191 @@
+  return {
+    __esModule: true,
+    sorobanServer: mockServer,
+    default: mockServer,
+  };
+});
+
+const { default: prismaMock } = (await import('../../src/config/prisma.js')) as any;
+const { sorobanServer } = (await import('../../src/indexer/sorobanClient.js')) as any;
+const { tick, startPolling, stopPolling, getCursor, resetPoller } = await import(
+  '../../src/indexer/poller.js'
+);
```

ESLint

[error] Replace `·await·import(⏎··'../../src/indexer/poller.js'⏎` with `⏎··await·import('../../src/indexer/poller.js'`

(prettier/prettier)